src/main/java/squid/tasks/Task.java

```diff
@@ -0,0 +1,68 @@
+package squid.tasks;
+
+/**
+ * Class encapsulating a Task.
+ */
+public abstract class Task implements TaskInterface {
+
+    protected boolean completed;
+
+    /**
+     * The name of the task.
+     */
+    private String taskName;
+
+    /**
+     * Initializer for task.
+     * @param taskName The name of the task.
+     */
+    public Task(String taskName) {
+        this.taskName = taskName;
+        this.completed = false;
+    }
+
+    /**
+     * Get task name.
+     */
+    public String getTaskName() {
+        return this.taskName;
+    }
+
+    /**
+     * Set the task to be either complete or incomplete.
+     *
+     * @param completed If the task is to be set as complete.
+     */
+    public void setCompleted(boolean completed) {
+        this.completed = completed;
+    }
+
+    /**
+     * @return Whether the task is complete.
+     */
+    public boolean isCompleted() {
```

Comment:
Nice that you followed the coding standard to make the method sound like booleans.